### ISSUE TITLE
New-HPOVOSDeploymentServer fails in 5.20

### ISSUE THREAD

**Issue by VincentBerger** [HewlettPackard/POSH-HPOneView#482]
### Actual Behavior
New-HPOVOSDeploymentServer fails with
```The field named mgmtNetworkUri near line 1, column 103 is not a valid field in this JSON object. Check the valid fields for this JSON object and remove or correct the unrecognized field.```
According to the API reference, mgmtNetworkUri should now be tucked into an imageStreamerData object

### Steps to reproduce

```PowerShell
VERBOSE: [SEND-HPOVREQUEST] Processing 'oneview52simu1.hp.local' appliance connection request. 1 of 1
VERBOSE: [SEND-HPOVREQUEST] Requested URI '/rest/deployment-servers' to 'oneview52simu1.hp.local'
VERBOSE: [SEND-HPOVREQUEST] Body object found. Converting to JSON.
VERBOSE: [SEND-HPOVREQUEST] HTTP Method is POST. Removing 'ApplianceConnection' NoteProperty from object(s).
VERBOSE: InputObject is [PSCustomObject]. Copying...
VERBOSE: [SEND-HPOVREQUEST] Request Body: {"description":"Image Streamer for Logical Enclosure 1","name":"LE1 Image Streamer","mgmtNetworkUri":"/rest/ethernet-networks/c7f51c11-54b5-44f2-bccc-c424330d3353","applianceUri":"/rest/deployment-servers/image-streamer-appliances/bffbb43f-9948-48ea-b920-8512933fdfc7","deplManagersType":"Image Streamer"}
VERBOSE: [SEND-HPOVREQUEST] Request: POST https://oneview52simu1.hp.local/rest/deployment-servers
VERBOSE: [SEND-HPOVREQUEST] Request Header 1: User-Agent = HPEOneview_PowerShell/5.20 (Microsoft Windows NT 10.0.14393.0)
VERBOSE: [SEND-HPOVREQUEST] Request Header 2: Content-Type = application/json
VERBOSE: [SEND-HPOVREQUEST] Request Header 3: Accept = application/json, text/html, application/xhtml+xml
VERBOSE: [SEND-HPOVREQUEST] Request Header 4: X-API-Version = 1600
VERBOSE: [SEND-HPOVREQUEST] Request Header 5: accept-language = en_US
VERBOSE: [SEND-HPOVREQUEST] Request Header 6: accept-encoding = gzip, deflate
VERBOSE: [SEND-HPOVREQUEST] Request Header 7: auth = [*****REDACTED******]
VERBOSE: [SEND-HPOVREQUEST] Net.WebException Error caught
VERBOSE: [SEND-HPOVREQUEST] Exception Object:
PSMessageDetails      :
Exception             : System.Net.WebException: The remote server returned an error: (400) Bad Request.
                           at System.Net.HttpWebRequest.GetResponse()
                           at CallSite.Target(Closure , CallSite , WebRequest )
TargetObject          :
CategoryInfo          : NotSpecified: (:) [], WebException
FullyQualifiedErrorId : WebException
ErrorDetails          :
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at Send-HPOVRequest<Process>, C:\Users\demopaq\Documents\PowerShell\Modules\HPOneView.520\5.20.2422.3962\HPOneView.520.psm1: line 5610
                        at New-HPOVOSDeploymentServer<Process>, C:\Users\demopaq\Documents\PowerShell\Modules\HPOneView.520\5.20.2422.3962\HPOneView.520.psm1: line 96168
                        at <ScriptBlock>, C:\Users\demopaq\Documents\PowerShell\Modules\HPOneView.520\5.20.2422.3962\HPOneView.520.psm1: line 95658
                        at Get-HPOVImageStreamerAppliance<Process>, C:\Users\demopaq\Documents\PowerShell\Modules\HPOneView.520\5.20.2422.3962\HPOneView.520.psm1: line 95654
                        at <ScriptBlock>, <No file>: line 1
PipelineIterationInfo : {}



VERBOSE: [SEND-HPOVREQUEST] Exception Message: The remote server returned an error: (400) Bad Request.
VERBOSE: [SEND-HPOVREQUEST] InnerException FullyQualifiedErrorId:
VERBOSE: [SEND-HPOVREQUEST] InnerException Message:
VERBOSE: [SEND-HPOVREQUEST] Exception
VERBOSE: [SEND-HPOVREQUEST] Getting Error Response
VERBOSE: [SEND-HPOVREQUEST] ERROR RESPONSE: {"errorCode":"UNRECOGNIZED_JSON_FIELD","message":"Unrecognized JSON field.",
"details":"The field named mgmtNetworkUri near line 1, column 103 is not a valid field in this JSON object.","messagePar
ameters":[],"recommendedActions":["Check the valid fields for this JSON object and remove or correct the unrecognized fi
eld."],"errorSource":null,"nestedErrors":[],"data":{}}
VERBOSE: [SEND-HPOVREQUEST] Response Status: HTTP 400 [Bad Request]
VERBOSE: [SEND-HPOVREQUEST] Response Header: Date = close
VERBOSE: [SEND-HPOVREQUEST] Response Header: Server = chunked
VERBOSE: [SEND-HPOVREQUEST] Response Header: Cache-Control = application/json; charset=utf-8
VERBOSE: [SEND-HPOVREQUEST] Response Header: Pragma = 0
VERBOSE: [SEND-HPOVREQUEST] Response Header: X-XSS-Protection =
VERBOSE: [SEND-HPOVREQUEST] Response Header: X-Frame-Options =
VERBOSE: [SEND-HPOVREQUEST] Response Header: X-Content-Type-Options =
VERBOSE: [SEND-HPOVREQUEST] Response Header: Connection =
VERBOSE: [SEND-HPOVREQUEST] Response Header: Transfer-Encoding =
VERBOSE: [SEND-HPOVREQUEST] Response Header: Content-Type =
VERBOSE: [SEND-HPOVREQUEST] Response Header: Expires =
VERBOSE: [SEND-HPOVREQUEST] HTTP 400 error caught.
VERBOSE: [NEW-ERRORRECORD] Building ErrorRecord object
VERBOSE: [SEND-HPOVREQUEST] Cleaning up HttpWebRequest
New-HPOVOSDeploymentServer: The field named mgmtNetworkUri near line 1, column 103 is not a valid field in this JSON object. Check the valid fields for this JSON object and remove or correct the unrecognized field.
```

### Version Information

```PowerShell
# Output from $PSLibraryVersion
oneview52simu1.hp.local             LibraryVersion Path
-----------------------             -------------- ----
ApplianceVersion: 5.20.00.419867.00 5.20.2422.3962 C:\Users\demopaq\Documents\PowerShell\Modules\hponeview.520\5.20.24…
```

**Comment by ChrisLynchHPE** [comment]
Thanks for reporting this.  It is due to an API change that I didn't pick up.  It'll be fixed in the next module update.

**Comment by PeterM-vLabs** [comment]
Hi Chris,

Any idea when the updated module will be available?  We are using the 5.2 libs for our Synergy automation in the vLabs and hit this bug this morning.

Thanks,
Peter

**Comment by ChrisLynchHPE** [comment]
@PeterM-vLabs , soon.  I'm in the process of doing dome final testing of a 5.20 library update.

**Comment by ChrisLynchHPE** [comment]
This has been fixed in the update I just published. Please grab the update from the PowerShell Gallery.

**Comment by VincentBerger** [comment]
Fix verified in 5.20.2452. Closing
